docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/ConvertToGrayScale.cs

```diff
@@ -42,29 +43,31 @@ public static void Example()
                            .Append(mlContext.Transforms.ConvertToGrayscale("Grayscale", "ImageObject"));
 
             var transformedData = pipeline.Fit(data).Transform(data);
+
             // The transformedData IDataView contains the loaded images column, and the grayscaled column.
+            PrintColumns(transformedData);
 
-            // Preview the transformedData. 
-            var transformedDataPreview = transformedData.Preview();
-            PrintPreview(transformedDataPreview);
             // ImagePath    Name         ImageObject            Grayscale
             // tomato.bmp   tomato       System.Drawing.Bitmap  System.Drawing.Bitmap
             // banana.jpg   banana       System.Drawing.Bitmap  System.Drawing.Bitmap
             // hotdog.jpg   hotdog       System.Drawing.Bitmap  System.Drawing.Bitmap
             // tomato.jpg   tomato       System.Drawing.Bitmap  System.Drawing.Bitmap
         }
 
-        private static void PrintPreview(DataDebuggerPreview data)
+        private static void PrintColumns(IDataView transformedData)
         {
-            foreach (var colInfo in data.ColumnView)
-                Console.Write("{0,-25}", colInfo.Column.Name);
-
-            Console.WriteLine();
-            foreach (var row in data.RowView)
+            var imagePathColumn = transformedData.GetColumn<string>("ImagePath").GetEnumerator();
+            var namePathColumn = transformedData.GetColumn<string>("Name").GetEnumerator();
+            var imageObjectColumn = transformedData.GetColumn<Bitmap>("ImageObject").GetEnumerator();
+            var grayScaleColumn = transformedData.GetColumn<Bitmap>("Grayscale").GetEnumerator();
```

Comment:
>grayScaleColumn [](start = 16, length = 15)

We shouldn't have written this code this way, but note that enumerators implement `IDisposable`...